tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go

```diff
@@ -392,13 +465,49 @@ func testScaledObjectErrors(t *testing.T, data templateData) {
 	time.Sleep(10 * time.Second)
 }
 
+func testScaledJobErrors(t *testing.T, data templateData) {
+	t.Log("--- testing scaled job errors ---")
+
+	KubectlDeleteWithTemplate(t, data, "scaledJobTemplate", scaledJobTemplate)
+	KubectlApplyWithTemplate(t, data, "wrongScaledJobTemplate", wrongScaledJobTemplate)
+
+	time.Sleep(20 * time.Second)
+
+	family := fetchAndParsePrometheusMetrics(t, fmt.Sprintf("curl --insecure %s", kedaOperatorCollectorPrometheusExportURL))
+	val, ok := family["keda_scaledjob_errors_total"]
```

Comment:
This isn't consistent with prometheus test case. There the metric name is `keda_scaled_job_errors`

Reply:
ditto